hazelcast/src/main/java/com/hazelcast/monitor/impl/NearCacheStatsImpl.java

```diff
@@ -127,13 +133,41 @@ public double getRatio() {
     }
 
     @Override
+    public long getEvictions() {
+        return evictions;
+    }
+
+    public void setEvictions(long evictions) {
```

Comment:
Is this operation not racy? So imagine 2 threads do this at the same time, then you could be suffering from a lost update.